backend/kaos_backend/util/tests/test_validators.py

```diff
@@ -17,6 +17,14 @@ def create_file(path):
     return temp
 
 
+def make_executable_file(path):
+    with open(path, 'w') as executable_file:
+        line = "#!"
+        executable_file.write(line)
+    executable_file.close()
```

Comment:
I think that you do not need `executable_file.close()`

Reply:
This entire function is not being used anymore